_posts/2016/2016-03-24-ecommerce-barcamp-jena.md

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
general comment: please use small file extensions, e.g. jpg instead of JPG
otherwise tests will fail.